Source/gui/SNTMessageView.swift

```diff
@@ -161,3 +162,19 @@ public func DismissButton(
   .keyboardShortcut(.escape, modifiers: .command)
   .help("⌘ Esc")
 }
+
+// AttributedText is like Text() but it supports all the features of NSAttributedString()
+// by using NSTextField under the hood.
```

Comment:
Does this mean we can use markdown here? 

Asking because I skimmed https://developer.apple.com/documentation/foundation/attributedstring/instantiating_attributed_strings_with_markdown_syntax and it looks like we might support it with this change.

Reply:
No, it doesn't. The actual message goes through SNTBlockMessage first where either HTML tags are stripped out or more HTML is added to make it look right. 

Markdown has to be initialized with a different initializer and it's hard to embed in a plist with newlines, so it would be much harder to use correctly than the HTML support we currently have.